en/docs/tutorials/integration-tutorials/service-catalog-tutorial-for-proxy-services.md

```diff
@@ -0,0 +1,518 @@
+
+    <img src="{{base_path}}/assets/img/integrate/tutorials/service-catalog/expose-soap-service/api-deployment-revision.png" alt="select gateways for the deployment">
+
+3.  **Optionally**, you can add a description.
+
+4.  Click **Deploy**. 
+
+You will now see the deployment as the first revision of the API:
+
+<img src="{{base_path}}/assets/img/integrate/tutorials/service-catalog/expose-soap-service/api-gateway-deployment-summary.png" alt="api first revision">
+
+### Step 8: Publish the API
+
+Go to the API overview in the **Publisher** portal and click **Publish** for the `StockQuoteProxy` as shown below.
+
+<img src="{{base_path}}/assets/img/integrate/tutorials/service-catalog/api-overview-publish.png">
+
+The API is now available in the **Developer** portal for consumers to access.
+
+### Step 9: Subscribe to the API
+
+Now, let's assume you are an API consumer who wants to use the API. As a consumer, you need to first subscribe to the API.
+
+1.  Sign in to the **Developer** portal: `https://localhost:9443/devportal/apis`. 
+
+    !!! Tip
+        Use `admin` as the username and password.
+
+2.  Go to the **API** tab. The `StockQuoteProxy` is listed as shown below.
+
+    <img src="{{base_path}}/assets/img/integrate/tutorials/service-catalog/expose-soap-service/developer-portal-api-list.png">
+
+3.  Select the `StockQuoteProxy` to open the API overview.
+
+    <img src="{{base_path}}/assets/img/integrate/tutorials/service-catalog/expose-soap-service/developer-portal-api-overview.png">
+
+4.  Go to the **Subscriptions** tab and subscribe using the **DefaultApplication** as shown below.
+
+    <img src="{{base_path}}/assets/img/integrate/tutorials/service-catalog/expose-soap-service/developer-portal-api-subscription.png">
+
+!!! Tip
+    For detailed instructions, see [Subscribe to an API]({{base_path}}/consume/manage-subscription/subscribe-to-an-api/).
+
+### Step 10: Use the SOAP Passthrough API
+
+!!! Info "Before you begin"
+
+    Let's start the back-end service.
+
+    1. Download the [back-end service](https://github.com/wso2-docs/WSO2_EI/blob/master/Back-End-Service/axis2Server.zip).
+    2. Extract the downloaded zip file.
+    3. Open a terminal, navigate to the `axis2Server/bin/` directory inside the extracted folder.
+    4. Execute the following command to start the axis2server with the SimpleStockQuote back-end service:
+    
+        ```bash tab='On MacOS/Linux/CentOS'
+        sh axis2server.sh
+        ```
+    
+        ```bash tab='On Windows'
+        axis2server.bat
+        ```
+
+**Generate access token**
+
+When you consume an API from the marketplace, your access to the API is authenticated. Therefore, the **DefaultApplication** that you used for subscribing to the API should get an access token for the gateway environment in which the API is deployed. Since the `StockQuoteProxy` is deployed in the Production gateway, you must generate **PROD** keys.
```

Comment:
```suggestion
When you consume an API from the marketplace, your access to the API is authenticated. Therefore, the **DefaultApplication** that you used for subscribing to the API should get an access token for the Gateway environment in which the API is deployed. Since the `StockQuoteProxy` is deployed in the Production Gateway, you must generate **PROD** keys.
```